scalr/README.md

```diff
@@ -0,0 +1,71 @@
+# Scalr Integration
+
+## Overview
+
+This integration collects metrics from Scalr.
+
+## Setup
+The Scalr integration is not included in the [Datadog Agent][2] package, so you need to install it.
+
+### Installation
+
+For Agent v7.21+ / v6.21+, follow the instructions below to install the Scalr integration on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.
+
+1. Run the following command to install the Agent integration:
+
+   ```shell
+   datadog-agent integration install -t datadog-scalr==<INTEGRATION_VERSION>
```

Comment:
The command needs to include the integration version. Can you replace <INTEGRATION_VERSION> with the integration version?